pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/PdfBuilder.java

```diff
@@ -76,6 +76,70 @@ public void updatePdfs(Handle handle) {
         }
     }
 
+    /**
+     * Update existing pdf templates (pdf bytes only) in the pdf config.
+     * Does not update the pdf config like sources/substitutions or new insertions
+     */
+    public void updatePdfTemplates(Handle handle, long studyId, Config pdfCfg, String pdfFilePath) {
```

Comment:
Can be used for any future pdf byte updates into DB.
Supports new versioned pdf configuration.
Can update ALL pdf templates in a config OR selected pdf file only in the config.